src/uiConfig/VizbeeUIConfiguration.js

```diff
@@ -0,0 +1,20 @@
+
+import { NativeModules } from "react-native";
+
+const VizbeeNativeManager = NativeModules.VizbeeNativeManager || {};
+
+/**
+* Configuration class for managing UI configurations in the Vizbee SDK.
+*/
+export default class VizbeeUIConfiguration {
+ 
+    /**
+     * Sets a card configuration for a specific card type.
+     * 
+     * @param {Object} cardConfiguration - The configuration to be set
+     * @param {string} cardType - The type of card this configuration applies to
+     */
+    setCardConfiguration(cardConfiguration, cardType) {
```

Comment:
@shivareddy Can we define the API as setCardConfigurationByType(cardType, cardConfiguration)?

Reply:
I recommend `setCardConfiguration` for the following :

1. It's more concise while still being clear
2. The 'byType' is redundant since the parameter name cardType already indicates this
3. Follows common naming patterns in standard libraries
4. Easier to read in actual code usage

Reply:
OK shiva